pkg/apis/kops/v1alpha3/topology.go

```diff
@@ -22,30 +22,27 @@ const (
 )
 
 type TopologySpec struct {
-	// The environment to launch the Kubernetes masters in public|private
-	Masters string `json:"masters,omitempty"`
+	// ControlPlane specifies the environment for launching the control plane nodes. (public, private)
+	ControlPlane string `json:"control-plane,omitempty"`
```

Comment:
Not quite sure we have any other place using `-`.
```suggestion
	ControlPlane string `json:"controlPlane,omitempty"`
```

Reply:
Fixed